spec/abstractops.html

```diff
@@ -719,26 +719,27 @@ <h1>
         _minute_: an integer,
         _second_: an integer,
         _subSecondNanoseconds_: an integer,
-        _precision_: an integer in the inclusive range 0 to 9, *"minute"*, or *"auto"*
+        _precision_: an integer in the inclusive range 0 to 9, *"minute"*, *"minute-legacy"*, or *"auto"*,
+        optional _style_: ~legacy~ or ~separated~,
```

Comment:
Would renaming these enumeration values be in scope? The ISO 8601 terms are "basic format" and "extended format", respectively. It's not necessary to adopt those per se, but "legacy" in particular seems misnamed.

Reply:
Sure! Maybe separated and unseparated? Or even a `_separator_` optional param?

Reply:
`_style_: ~separated~ or ~unseparated~` works for me, but `_separator_` is a bit too generic.

Reply:
> `_style_: ~separated~ or ~unseparated~` works for me

Sold!  For consistency, I'll also make it optional in both FormatTimeString and FormatOffsetTimeZoneIdentifier.